integrations/skopt/tests/Neptune-Skopt_upgraded_libs.py

```diff
@@ -4,7 +4,7 @@
 
 ## Install dependencies
 
-get_ipython().system(" pip install --quiet scikit-optimize==0.8.1 neptune-client==0.4.132 neptune-contrib['monitoring']==0.25.0")
+get_ipython().system(" pip install --quiet lightgbm==2.2.3 scikit-optimize==0.8.1 neptune-client==0.4.132 neptune-contrib['monitoring']==0.25.0")
 
 get_ipython().system(" pip install --quiet scikit-optimize neptune-client neptune-contrib['monitoring'] --upgrade")
```

Comment:
@szymon-kuklewicz the idea of this second install line is to upgrade those libs and see if it fails.